workers/unity/Packages/com.improbable.gdk.core/Attributes/RemoveAtEndOfTickAttribute.cs

```diff
@@ -8,7 +8,7 @@ namespace Improbable.Gdk.Core
     ///     Can be added to components extending <see cref="IComponentData" /> or <see cref="ISharedComponentData" />
     /// </summary>
     [AttributeUsage(AttributeTargets.Struct)]
-    public class RemoveAtEndOfTickAttribute : Attribute
+    public class RemoveAtEndOfTick : Attribute
```

Comment:
Re my previous comment. We can still call this `RemoveAtEndOfTickAttribute`, however when we annotate a class with it we can just use `[RemoveAtEndOfTick]`. This is just a C# convention.

Reply:
Interesting, didn't know about that C# feature, will revert this change